ui/lib/pki/addon/components/pki-generate-root.hbs

```diff
@@ -1,69 +1,126 @@
-<form {{on "submit" (perform this.save)}} data-test-pki-config-generate-root-form>
-  <MessageError @errorMessage={{this.errorBanner}} class="has-top-margin-s" />
-  <h2 class="title is-size-5 has-border-bottom-light page-header" data-test-generate-root-title="Root parameters">
-    Root parameters
-  </h2>
-  {{#each this.defaultFields as |field|}}
-    {{#let (find-by "name" field @model.allFields) as |attr|}}
-      <FormField @attr={{attr}} @model={{@model}} @modelValidations={{this.modelValidations}} data-test-field>
-        {{#if (eq field "customTtl")}}
-          {{! customTtl attr has editType yield, which will render this }}
-          <PkiNotValidAfterForm @attr={{attr}} @model={{@model}} />
+{{! Show results if model has an ID, which is only generated after save }}
+{{#if @model.id}}
+  <Toolbar />
+  {{#if @model.privateKey}}
+    <div class="has-top-margin-m">
+      <AlertBanner
+        @title="Next steps"
+        @type="warning"
+        @message="This private key material will only be available once. Copy or download it now."
```

Comment:
I don't think there's a download action for the toolbar here, right?

Reply:
Good catch! I'll update to match the CSR language (not sure where I got this 🤔 )

Reply:
I think we have a banner that says this for the key view? There is somewhere I remember adding a download option! 😂